crates/bevy_math/Cargo.toml

```diff
@@ -23,6 +23,7 @@ approx = "0.5"
 # Supply rngs for examples and tests
 rand = "0.8"
 rand_chacha = "0.3"
+bevy_math = { path = ".", version = "0.14.0-dev", features = ["approx"] }
```

Comment:
```suggestion
# Enable the approx feature when testing.
bevy_math = { path = ".", version = "0.14.0-dev", features = ["approx"] }
```

Given how cursed this fix is, I think it deserves a comment explaining what it does.